payments-core/src/test/java/com/stripe/android/view/CardInputWidgetTest.kt

```diff
@@ -1770,6 +1770,18 @@ internal class CardInputWidgetTest {
         )
     }
 
+    @Test
+    fun `Re-fetches card brands when first eight are deleted and re-entered`() = runCardInputWidgetTest(
+        true
+    ) {
+        cardNumberEditText.setText("4000 0025 0000 1001")
+        assertThat(cardBrandView.possibleBrands.size).isGreaterThan(0)
+        updateCardNumberAndIdle("0000 1001")
+        assertThat(cardBrandView.possibleBrands.size).isEqualTo(0)
+        cardNumberEditText.setText("4000 0025 0000 1001")
+        assertThat(cardBrandView.possibleBrands.size).isGreaterThan(0)
```

Comment:
should this be asserting that there are 2 possible brands?

Reply:
Good call, updated 👍